extensions/blocks/markdown/index.js

```diff
@@ -67,4 +67,14 @@ export const settings = {
 	edit,
 
 	save,
+
+	example: {
+		attributes: {
+			source:
+				'## ## Try Markdown\n\n' +
+				'Markdown is a text formatting syntax that is converted into HTML. ' +
+				'You can _emphasize_ text or **make it strong** with just a few characters.\n\n' +
+				'Search up ["Markdown"](https://daringfireball.net/projects/markdown/) to learn more.',
```

Comment:
What do you think about linking to our support doc instead?
https://en.support.wordpress.com/markdown-quick-reference/

Reply:
I was planning to remove the link altogether since it's impossible to click.

Reply:
I changed my mind. I'm going to leave the link there because it's still useful in the Block Inspector.